src/vs/workbench/parts/terminal/electron-browser/terminalLinkHandler.ts

```diff
@@ -76,8 +109,8 @@ export class TerminalLinkHandler {
 			this._handleLocalLink(url);
 			return;
 		});
+
 		return this._xterm.registerLinkMatcher(this._localLinkRegex, wrappedHandler, {
-			matchIndex: 1,
```

Comment:
But `matchIndex` is still generally supported right? Custom link matchers need to be able to specify it still.